packages/firebase_ml_custom/example/README.md

```diff
@@ -1,16 +1,20 @@
 # Example of Firebase Machine Learning Custom
-
 Demonstrates how to use the firebase_ml_custom plugin.
 
 ## Usage
+In order to run this example app you first need to **upload a tflite model to the Firebase console.**
+This app uses `mobilenet_v1_1.0_224.tflite` label recognition model, which you can find in the `assets` folder.
+###### How to host this model on Firebase:
+1. In the `Machine Learning` section of the Firebase console, click the `Custom` tab.
+2. Click `Add model`.
+3. Specify a name that will be used to identify your model in your Firebase project, `mobilenet_v1_1_0_224` for this example app. Then upload the TensorFlow Lite model file.
 
-This example uses the *image_picker* plugin to get images from the device gallery. If using an iOS
-device you will have to configure your project with the correct permissions seen under iOS
-configuration [here](https://pub.dartlang.org/packages/image_picker).
+Remember to replace `google-services.json` and `GoogleService-Info.plist` with those from the project with the uploaded model.
```

Comment:
Not sure what other example apps in flutter say in their README, but should there be a first step of setting up firebase for your project? https://firebase.google.com/docs/flutter/setup

Reply:
Readmes in other plugins' examples:
- https://github.com/PolinaGo/flutterfire/tree/master/packages/firebase_storage/example
- https://github.com/PolinaGo/flutterfire/tree/master/packages/firebase_remote_config/example
- https://github.com/PolinaGo/flutterfire/tree/master/packages/firebase_admob/example

I think information about how to set up firebase should be included in Flutterfire root Readme instead.